doc/enterprise-shield/p_enterprise_shield_config.md

```diff
@@ -0,0 +1,418 @@
+
+Step 2: Before You Begin
+----------------------------------------------
+
+This walkthrough covers Use Case 3: Maximum (End-to-End) Transport Security from the set of [Common Enterprise Shield™ Use Cases](c_enterprise_shield_use_cases.md#usecase3) because that is the most secure example of Enterprise Shield™. By the end of this walkthrough, you will have the foundational knowledge required to understand and configure any of the Enterprise Shield™ use cases.
+
+Step 3: Download and Set Up KAAZING Gateway
+---------------------------------------------------------------
+
+Download and set up two KAAZING Gateways, with one Gateway in the DMZ and one in the internal trusted network. This is described in [Setting Up the Gateway](../about/setup-guide.md).
+
+Step 4: Configure the Internal Gateway
+-------------------------------------------------------------------
+
+Perform the following steps to set up the App1 and App2 services on the internal Gateway.
+
+**Notes:**
+-   The examples in this walkthrough use the `.net` domain (such as `tcp://gateway.example.net:8080`) to indicate internal, nonpublic URLs, and use the `.com` domain to indicate public URLs. All domains and URLs are for example purposes only. Wherever a URI uses `example` in the domain name, you should replace that with your own company name.
+-   Network security is improved by using separate NICs (Network Interface Cards) for the [frontplane](http://developer.kaazing.com/documentation/kaazing-glossary.html#frontplane-and-backplane) and [backplane](http://developer.kaazing.com/documentation/kaazing-glossary.html#frontplane-and-backplane). Although it's not necessary to have multiple NICs for Enterprise Shield™ to work, this walkthrough assumes separate frontplane and backplane NICs. The configuration defines default values for these interfaces once in the properties element using easily identifiable names and the values are propagated throughout the entire configuration when the Gateway starts. See [Documentation Conventions](../about/about.md) for more information about variables.
+
+
+The following snippet shows the App1 Service configuration in the `gateway-config.xml` file.
+
+``` xml:
+<properties>
+  <name>dmz.backplane.hostname</name>
+  <value>gateway.example.dmz.net</value>
+</properties>
+
+<service>
+
+  <name>Internal App1</name>
+  <accept>wss://gateway.example.com:8443/app1</accept>
+  <connect>tcp://127.0.0.1:3101</connect>
+
+  <type>proxy</type>
+
+  <accept-options>
+    <http.transport>socks://${dmz.backplane.hostname}:1080</http.transport>
+    <socks.mode>reverse</socks.mode>
+    <socks.retry.maximum.interval>1 second</socks.retry.maximum.interval>
+    <socks.transport>
```

Comment:
I think this wrap was coded into the source because we need to wrap for the 4.0 doc (HTML) but we don't need to wrap for the 5.0 doc (MD).  Anyway, that is the reasoning for the wrap.  Thanks.